CMakeLists.txt

```diff
@@ -110,14 +109,22 @@ generate_messages(
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-        INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/src/MusicBeatDetector/MusicBeatDetector/include
+        INCLUDE_DIRS
+                ${CMAKE_CURRENT_SOURCE_DIR}/src/MusicBeatDetector/MusicBeatDetector/include
                 ${CMAKE_CURRENT_SOURCE_DIR}/src/MusicBeatDetector/3rdParty/armadillo-code/include
                 ${CMAKE_CURRENT_SOURCE_DIR}/src/MusicBeatDetector/3rdParty/fftw/build/include
-        LIBRARIES MusicBeatDetector ${openblas_LIBRARY} ${fftw_LIBRARY}
+        LIBRARIES
+                MusicBeatDetector
+                # Needs to be hardcoded as the variables are not set until the call to add_subdirectory
+                ${CMAKE_CURRENT_SOURCE_DIR}/src/MusicBeatDetector/3rdParty/OpenBLAS/libopenblas.a
```

Comment:
Not sure about this. Should this be CMAKE_CURRENT_BINARY_DIR ? @philippewarren  @mamaheux